### ISSUE TITLE
Game Activities crashing frequently

### ISSUE THREAD

**Issue by Stuke00** [Lacro59/playnite-gameactivity-plugin#166]
[GameActivities.zip](https://github.com/Lacro59/playnite-gameactivity-plugin/files/9639633/GameActivities.zip)

Added zip file that Playnite keeps prompting me to send to the developer.

**Comment by deangelnet** [comment]
I have 10 games that game activities crash since playnite V9.
1. [warcraft III.zip](https://github.com/Lacro59/playnite-gameactivity-plugin/files/9639765/warcraft.III.zip)
2. [toweroffantasy.zip](https://github.com/Lacro59/playnite-gameactivity-plugin/files/9639766/toweroffantasy.zip)
3. [honkaiimpact.zip](https://github.com/Lacro59/playnite-gameactivity-plugin/files/9639767/honkaiimpact.zip)

Will add other games when played them again.

**Comment by Stuke00** [comment]
Did you mean playnite 10? That just came out recently. Sent from my iPhoneOn Sep 24, 2022, at 19:00, De Angel Net ***@***.***> wrote:﻿
I have 10 games that game activities crash since playnite V9.

warcraft III.zip
toweroffantasy.zip
honkaiimpact.zip

Will add other games when played them again.

**Comment by deangelnet** [comment]
> Did you mean playnite 10? That just came out recently. Sent from my iPhoneOn Sep 24, 2022, at 19:00, De Angel Net ***@***.***> wrote:﻿ I have 10 games that game activities crash since playnite V9. warcraft III.zip toweroffantasy.zip honkaiimpact.zip Will add other games when played them again. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

Since playnite 9 and 10. Have the same error on some games.

**Comment by MistahZed** [comment]
I updated today, was crashing until I removed my old data. Then was fine until I tried to log the hardware stats. First time exiting a game where I was tracking stats Playnite will crash repeatedly until I remove the data with hardware stats again. Logs attached from when it gave me the option.
[gameactivity log.zip](https://github.com/Lacro59/playnite-gameactivity-plugin/files/9640589/gameactivity.log.zip)
[gameactivity_log2.zip](https://github.com/Lacro59/playnite-gameactivity-plugin/files/9640590/gameactivity_log2.zip)
